server/ladder_service.py

```diff
@@ -55,17 +71,26 @@ def __init__(self, database: FAFDatabase, games_service: GameService):
 
         asyncio.ensure_future(self.queues[queue_name].search(search))
```

Comment:
As far as I can tell `MatchmakerQueue::search()` parameters and docs are all correct. We need to use `ensure_future` because `search` will block until a match is found or the search is cancelled, and we need to keep processing messages while this happens.